### PR TITLE
Removing fake integration

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Removes a fake integration to enhance the custom log collection section of the Log doc.

What's is happening is we currently have 2 main ways to collect windows event

1 - with [win32 integration](https://github.com/DataDog/integrations-core/blob/master/win32_event_log/README.md)

2- with windows channel log collection (this PR)

the first one send windows event as events in DD events stream

the second one sends windows event as logs in DD log management solution.

### Motivation

Customers were confused with the duplicated content.

### Preview link

* https://docs-staging.datadoghq.com/gus/windows-event-collection/logs/log_collection/?tab=windowsevents

### CODE REVIEW

File: content/logs/log_collection/_index.md [hunk 1]
```diff
@@ -53,32 +54,15 @@ Datadog Agent v6 can collect logs from files or the network (TCP or UDP) and for
...
-| `sourcecategory` | optional  | Multiple value attribute. Can be used to refine the source attribtue. Example: `source:mongodb, sourcecategory:db_slow_logs`                                                                                                                                                                                               |
-| `tags`           | optional  | Add tags to each log collected, [learn more about tagging][13].                                                                                                                                                                                                                                                            |
-
-Finally [restart your Agent][10] to take into account this new configuration. You can then [run the Agent's `status` subcommand][11] to check if your custom configuration is correct.
+3. Add a custom log collection configuration group with the parameters below.
+4. Finally [restart your Agent][10] to take into account this new configuration.
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: It's not the final step; remove "Finally".

File: content/logs/log_collection/_index.md [hunk 2]
```diff
@@ -53,32 +54,15 @@ Datadog Agent v6 can collect logs from files or the network (TCP or UDP) and for
...
-| `sourcecategory` | optional  | Multiple value attribute. Can be used to refine the source attribtue. Example: `source:mongodb, sourcecategory:db_slow_logs`                                                                                                                                                                                               |
-| `tags`           | optional  | Add tags to each log collected, [learn more about tagging][13].                                                                                                                                                                                                                                                            |
-
-Finally [restart your Agent][10] to take into account this new configuration. You can then [run the Agent's `status` subcommand][11] to check if your custom configuration is correct.
+3. Add a custom log collection configuration group with the parameters below.
+4. Finally [restart your Agent][10] to take into account this new configuration. 
+5. Optionally [run the Agent's `status` subcommand][11] to check if your custom configuration is correct.
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: Why is this optional? It's good to verify things.

File: content/logs/log_collection/_index.md [hunk 3]
```diff
@@ -131,38 +109,76 @@ logs:
...
       - sshd.service
 ```
-
 [9]: /agent/faq/agent-configuration-files/
 
 {{% /tab %}}
 {{% tab "Windows Events" %}}
 
-Set `type` to **windows_event** then specify the `channel_path` of your Windows channel.
+Edit the file `win32_event_log.d/conf.yaml` in the Agent's `conf.d/` directory (or in the Datadog Agent Manager) with the channels from which you want to send Windows Events as log.
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: Can we just refer to the full path? Something like:
> Add the channels from which logs will be sent to `conf.d/win32_event_log.d/conf.yaml` manually, or via the Datadog Agent Manager.

File: content/logs/log_collection/_index.md [hunk 4]
```diff
@@ -131,38 +109,76 @@ logs:
...
 [9]: /agent/faq/agent-configuration-files/
 
 {{% /tab %}}
 {{% tab "Windows Events" %}}
 
-Set `type` to **windows_event** then specify the `channel_path` of your Windows channel.
+Edit the file `win32_event_log.d/conf.yaml` in the Agent's `conf.d/` directory (or in the Datadog Agent Manager) with the channels from which you want to send Windows Events as log. 
 
-Example: 
+To see the channel list run the following command in a powershell:
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: "PowerShell" - it's a proper name and is rendered in CamelCase.

File: content/logs/log_collection/_index.md [hunk 5]
```diff
@@ -131,38 +109,76 @@ logs:
...
 
-[Refer to the dedicated widows event documentation page][15].
+```
+Get-WinEvent -ListLog *
+```
 
-[15]: https://docs.datadoghq.com/integrations/windows_event_log/#log-collection
+To see the most active ones run the following command:
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: Active whats?

File: content/logs/log_collection/_index.md [hunk 6]
```diff
@@ -131,38 +109,76 @@ logs:
...
+    sourcecategory: windowsevent
+
+  - type: windows_event
+    channel_path: <CHANNEL_2>
+    source: <CHANNEL_2>
+    service: myservice
+    sourcecategory: windowsevent
+```
+
+Edit the `<CHANNEL_X>` parameters with the wanted channel to collect events from. Set the `source` to the channel name to benefit from the integration automatic processing Pipeline setup.
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: > Add the channels from which logs will be collected by editing the `<CHANNEL_X>` parameters.

> Set `source` to the …

I'm confused about this line - as in, I don't know what it means:
> integration automatic processing Pipeline setup.

File: content/logs/log_collection/_index.md [hunk 7]
```diff
@@ -131,38 +109,76 @@ logs:
...
+
+Finally [Restart the agent][1]
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: Comma after "Finally" (or remove "Finally").

File: content/logs/log_collection/_index.md [hunk 8]
```diff
@@ -131,38 +109,76 @@ logs:
...
+[1]: /agent/basic_agent_usage/windows/
+
+{{% /tab %}}
 {{< /tabs >}}
 
+List of all available parameters for log collection: 
+
+| Parameter        | Presence  | Description                                                                                                                                                                                                                                                                                                                |
+| ------           | -------   | ------                                                                                                                                                                                                                                                                                                                     |
+| `type`           | mandatory | Type of the log input source, it can be **tcp**, **udp**, **file**, **windows_event**, **docker**, or **journald**                                                                                                                                                                                                         |
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: > Type of log input source. Valid values are: `tcp`, `udp`, `file`, `windows_event`, `docker`, or `journald`.

File: content/logs/log_collection/_index.md [hunk 9]
```diff
@@ -131,38 +109,76 @@ logs:
...
+| `port`           | mandatory | If `type` is **tcp** or **udp**, port to listen log from.                                                                                                                                                                                                                                                                  |
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: Basically this entire list of `**`'d strings should be `back-ticked`…

File: content/logs/log_collection/_index.md [hunk 10]
```diff
@@ -131,38 +109,76 @@ logs:
...
+| `path`           | mandatory | If `type` is **file** or **journald**, path of the file to tail log from.                                                                                                                                                                                                                                                                  |
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: Not sure what "to tail log from" means. 😕

File: content/logs/log_collection/_index.md [hunk 11]
```diff
@@ -131,38 +109,76 @@ logs:
...
+| `channel_path`   | mandatory | If `type` is **windows_event**, list of windows event channel to collect.                                                                                                                                                                                                                                                  |
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: channels (plural)

File: content/logs/log_collection/_index.md [hunk 12]
```diff
@@ -131,38 +109,76 @@ logs:
...
+| `service`        | mandatory | Name of the service owning the log. If you instrumented your service with [Datadog APM][12], this should be the same service name.                                                                                                                                                                                         |
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: Should? As in it might not be? Let's be clear here.

File: content/logs/log_collection/_index.md [hunk 13]
```diff
@@ -131,38 +109,76 @@ logs:
...
+| `source`         | mandatory | Attribute that defines which integration is sending the logs. If the logs do not come from an existing integration then this field may include a custom source name. But we recommend matching this value to the namespace of any related [custom metrics][2] you are collecting, e.g, `myapp` from `myapp.request.count`) |
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: We don't say "we", remember? 😉

File: content/logs/log_collection/_index.md [hunk 14]
```diff
@@ -131,38 +109,76 @@ logs:
...
+| `include_units`  | optional  | If `type` is **journald**, list of specific journald units to includes.                                                                                                                                                                                                                                                    |
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: include (no s).

File: content/logs/log_collection/_index.md [hunk 15]
```diff
@@ -131,38 +109,76 @@ logs:
...
+| `exclude_units`  | optional  | If `type` is **journald**, list of specific journald units to exclude.                                                                                                                                                                                                                                                     |
+| `sourcecategory` | optional  | Multiple value attribute. Can be used to refine the source attribtue. Example: `source:mongodb, sourcecategory:db_slow_logs`                                                                                                                                                                                               |
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: typo: attribute

File: content/logs/log_collection/_index.md [hunk 16]
```diff
@@ -131,38 +109,76 @@ logs:
...
+| `tags`           | optional  | Add tags to each log collected, [learn more about tagging][13].                                                                                                                                                                                                                                                            |
```
Comment (phrawzty) [inline content/logs/log_collection/_index.md]: > … log collected. Learn more …

File: content/logs/log_collection/_index.md (hunk 9 above)
Comment (l0k0ms) [reply content/logs/log_collection/_index.md]: No only the parameter are between backquotes, parameter values are in bold to be consistent with the API doc https://docs.datadoghq.com/api/?lang=python#post-an-event

File: content/logs/log_collection/_index.md [hunk 17]
```diff
@@ -129,40 +107,80 @@ logs:
...
+{{% tab "Stream logs from journald" %}}
 
 To gather logs from JournalD, create a `journald.d/conf.yaml` file at the root of your [Agent's configuration directory][9] with the following content:
 
-```
+```yaml
 logs:
   - type: journald
     path: /var/log/journal/
     include_units:
```
Comment (NBParis) [inline content/logs/log_collection/_index.md]: Let's remove the include_units section for this doc. This example is covered in the specific journald integration documentation.

**Pull request merged**